File: backend/tank-battle-arena-91777/android_frontend/app/src/test/java/com/example/tankbattlearena/AllUnitTests.java
package com.example.tankbattlearena;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;

/**
 * Ensures at least one JUnit4 test class is discovered by Gradle.
 */
@RunWith(Suite.class)
@Suite.SuiteClasses({
        ExampleUnitTest.class
})
public class AllUnitTests {
}

File: backend/tank-battle-arena-91777/android_frontend/app/src/test/java/com/example/tankbattlearena/AllUnitTests.kt
package com.example.tankbattlearena

import org.junit.runner.RunWith
import org.junit.runners.Suite

@RunWith(Suite::class)
@Suite.SuiteClasses(
    ExampleUnitTest::class,
    AnotherUnitTest::class,
    JavaUnitTest::class
)
class AllUnitTests
